components/speedreader/rust/lib/src/readability/src/extractor.rs

```diff
@@ -49,6 +49,11 @@ lazy_static! {
     static ref JSONLD_SCHEMA: Regex = Regex::new(r#"^https?://schema\.org[/?\w/?]*$"#).unwrap();
 }
 
+static SHOW_ORIGINAL_DIV_ID: &str = "c93e2206-2f31-4ddc-9828-2bb8e8ed940e";
+static READ_TIME_DIV_ID: &str = "da24e4ef-db57-4b9f-9fa5-548924fc9c32";
+static META_DATA_AREA_DIV_ID: &str = "3bafd2b4-a87d-4471-8134-7a9cca092000";
+static MAIN_CONTENT_DIV_ID: &str = "7c08a417-bf02-4241-a55e-ad5b8dc88f69";
```

Comment:
what's the purpose of these DIV_IDs? what would happen if a page itself set these as div ids?

Reply:
These ids used in speedreader-desktop.js to execute some post-load logic.

Reply:
Nothing special will happen, because these ids at top of the DOM (and this part of DOM fully generated by our engine).